### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by giscus** [giscus/giscus#1223] [General]
# Welcome to giscus!

http://localhost:3001/shut/15102023

**Comment by nschonni** [comment]
@laymonage it may be time to disallow using this repo for the hosted bot. We keep getting people using it for testing, which is annoying to any repo watchers. @yeargun, this is annoying, don't do testing on peoples projects

**Reply by yeargun** [reply]
Pardon me man. I wasn't aware of it.
